include/z64environment.h

```diff
@@ -30,6 +30,14 @@ typedef enum {
     /* 13 */ SKYBOX_DMA_PAL2_START
 } SkyboxDmaState;
 
+typedef enum {
+    /* 0 */ SANDSTORM_OFF,
+    /* 1 */ SANDSTORM_FILL,
+    /* 2 */ SANDSTORM_UNFILL,
+    /* 2 */ SANDSTORM_ACTIVE,
+    /* 2 */ SANDSTORM_DISSIPATE
+} SandStormState;
```

Comment:
```suggestion
} SandstormState;
```
Is sandstorm 1 word?

Reply:
it is yeah, thanks